### PR TITLE
sui: `token_bridge` code formatting

### PR DESCRIPTION
**Pull request by optke3**
## Updates
- follow the convention of using `///` before function, struct, const error, and module definitions, and `//` inside of functions and in other places
- in `native_asset` destructor, just write `custody` instead of `custody: custody`
- regarding capitalization, follow the convention in the [Sui core library](https://github.com/MystenLabs/sui/tree/main/crates/sui-framework/sources) of always using caps lock and periods
- keep lines to 80 chars in length whenever possible

### CODE REVIEW

**Review by gator-boi — APPROVED** [review]
lgtm

File: sui/token_bridge/sources/governance/register_chain.move [hunk 1]
```diff
@@ -17,9 +17,9 @@ module token_bridge::register_chain {
     const E_INVALID_TARGET: u64 = 2;
 
     struct RegisterChain has copy, drop {
-        /// Chain ID
+        // Chain ID
```
[the remark this answers is not in the document]
Comment (optke3) [reply sui/token_bridge/sources/governance/register_chain.move:20]: what do you think about use of capitalization / periods?

Some comments are not complete sentences, and are instead just clauses or sentence fragments. Should we use caps / period in these cases or no caps / no periods?

[the remark this answers is not in the document]
Comment (optke3) [reply sui/token_bridge/sources/governance/register_chain.move:20]: Should we do
```
 /// Asserts that the VAA is from a known token bridge.
    public fun verify_emitter(token_bridge_state: &State, vm: &VAA) {
     ...
    }
```
or 
```
 /// asserts that the VAA is from a known token bridge
    public fun verify_emitter(token_bridge_state: &State, vm: &VAA) {
     ...
    }
```

[the remark this answers is not in the document]
Comment (optke3) [reply sui/token_bridge/sources/governance/register_chain.move:20]: Nevermind, will probably go with the Sui codebase convention of always using caps + periods.

**Review by a5-pickle — APPROVED** [review]

**Pull request merged**